### PR TITLE
8.0 azure theme text & link contrast fix - light theme only

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
## Previous Behavior

Low vision people and people sensitive to colors will find it difficult to recognize the control and differentiate between the surrounding text and link if the luminosity ratio is less than 3:1. Solution is to change body text in light theme only to #292827 to bring contrast ratio to 3:1. 

Encountered high contrast dark theme visual bug for link while working on this PR: on hover, link turns black, which blends in with background color. 
Link rest
image

Link hovered (turns black)
image

## New Behavior

1.) Updated the primary text color (light theme only) to #292827 to bring contrast ratio to 3.0 when used with inline <Link> in order to bring parity to bug found in KO controls. We are not updating text universally (for example, <Label/> control) because KO will not be doing it across their controls either. 
image

2.) Updated the color of links in message bar (Infobox in KO) <Link> colors updated for light theme only, maintained in other themes.

Rest link in message bar
image

Hover link in message bar
image

3.) To fix HCD link color, I aligned link hover to that of the portal, which is the same as rest (#) 
Link rest
image

Link hovered (no color change)
image

Helpful table:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bd15496050409ea20867d894f0841d7a8c1cd2a0'>bd15496050409ea20867d894f0841d7a8c1cd2a0</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/azure-themes/src/azure/AzureColors.ts [hunk 1]
```diff
@@ -177,7 +177,10 @@ export const DarkSemanticColors: IAzureSemanticColors = {
     placeholder: BaseColors.GRAY_A19F9D,
```
Comment (fabricteam) [inline packages/azure-themes/src/azure/AzureColors.ts:177]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27947&commitId=5c3146eba61133aa08fb531f7659fe215188c9e6&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by Jacqueline-ms** [comment]
Closed https://github.com/microsoft/fluentui/pull/27882 and re did here to resolve a branch issue.

**Review by robtaft-ms — APPROVED** [review]
Thanks!

**Pull request merged**